diff --git a/node_modules/@gorhom/bottom-sheet/lib/commonjs/hooks/useBoundingClientRect.js b/node_modules/@gorhom/bottom-sheet/lib/commonjs/hooks/useBoundingClientRect.js
index b4a90b7..c3e41f0 100644
--- a/node_modules/@gorhom/bottom-sheet/lib/commonjs/hooks/useBoundingClientRect.js
+++ b/node_modules/@gorhom/bottom-sheet/lib/commonjs/hooks/useBoundingClientRect.js
@@ -46,18 +46,22 @@ function useBoundingClientRect(ref, handler) {
     }
 
     // @ts-ignore 👉 https://github.com/facebook/react/commit/53b1f69ba
-    if (ref.current.unstable_getBoundingClientRect !== null) {
-      // @ts-ignore https://github.com/facebook/react/commit/53b1f69ba
-      const layout = ref.current.unstable_getBoundingClientRect();
-      handler(layout);
-      return;
+    if (ref.current.unstable_getBoundingClientRect !== null && ref.current.unstable_getBoundingClientRect !== undefined) {
+      if (typeof ref.current.unstable_getBoundingClientRect === 'function') {
+        // @ts-ignore https://github.com/facebook/react/commit/53b1f69ba
+        const layout = ref.current.unstable_getBoundingClientRect();
+        handler(layout);
+        return;
+      }
     }
 
     // @ts-ignore once it `unstable_getBoundingClientRect` gets stable 🤞.
-    if (ref.current.getBoundingClientRect !== null) {
-      // @ts-ignore once it `unstable_getBoundingClientRect` gets stable.
-      const layout = ref.current.getBoundingClientRect();
-      handler(layout);
+    if (ref.current.getBoundingClientRect !== null && ref.current.getBoundingClientRect !== undefined) {
+      if (typeof ref.current.getBoundingClientRect === 'function') {
+        // @ts-ignore once it `unstable_getBoundingClientRect` gets stable.
+        const layout = ref.current.getBoundingClientRect();
+        handler(layout);
+      }
     }
   });
 }
diff --git a/node_modules/@gorhom/bottom-sheet/src/hooks/useBoundingClientRect.ts b/node_modules/@gorhom/bottom-sheet/src/hooks/useBoundingClientRect.ts
index cc85c8c..8166b92 100644
--- a/node_modules/@gorhom/bottom-sheet/src/hooks/useBoundingClientRect.ts
+++ b/node_modules/@gorhom/bottom-sheet/src/hooks/useBoundingClientRect.ts
@@ -56,7 +56,7 @@ export function useBoundingClientRect(
     }
 
     // @ts-ignore 👉 https://github.com/facebook/react/commit/53b1f69ba
-    if (ref.current.unstable_getBoundingClientRect !== null) {
+    if (typeof ref.current.unstable_getBoundingClientRect === "function") {
       // @ts-ignore https://github.com/facebook/react/commit/53b1f69ba
       const layout = ref.current.unstable_getBoundingClientRect();
       handler(layout);
@@ -64,7 +64,7 @@ export function useBoundingClientRect(
     }
 
     // @ts-ignore once it `unstable_getBoundingClientRect` gets stable 🤞.
-    if (ref.current.getBoundingClientRect !== null) {
+    if (typeof ref.current.getBoundingClientRect === "function") {
       // @ts-ignore once it `unstable_getBoundingClientRect` gets stable.
       const layout = ref.current.getBoundingClientRect();
       handler(layout);
